design-library/source/_patterns/03-organisms/02-header/header.twig

```diff
@@ -0,0 +1,8 @@
+<header id="site-header" class="grid">
+  <div class="grid-cell grid-cell--logo">
+    {% include "@atoms/logo.twig"  %}
+  </div>
+  <div class="grid-cell grid-cell--social-icons">
+    {% include "@atoms/social-icons.twig"  %}
```

Comment:
so, ideally these are parameterized based on the mapping of values passed to this. The logo may be static (it's my logo), but the social icons i may want to wire into Drupal and map a list of values here. Can you please refactor this file to use the "with" utility to pass the "social media" JSON data you are already passing to the header?